gravitee-apim-cypress/cypress/model/apis.ts

```diff
@@ -220,3 +224,239 @@ export enum ApiPrimaryOwnerType {
   USER = 'USER',
   GROUP = 'GROUP',
 }
+export interface UpdateApiEntity {
+  name: string;
+  version: string;
+  description: string;
+  contextPath?: string;
+  services?: Services;
+  resources?: Resource[];
+  visibility: ApiVisibility;
+  tags?: string[];
+  picture?: string;
+  categories?: string[];
+  labels?: string[];
+  groups?: string[];
+  metadata?: any;
+  background?: string;
+  proxy?: Proxy;
+  paths?: Rule[];
+  flows?: Flow[];
+  plans?: Plan[];
+  properties?: PropertyEntity[];
+  gravitee?: string;
+  flow_mode?: FlowMode;
+  picture_url?: string;
+  path_mappings?: string[];
+  response_templates?: {};
+  lifecycle_state?: ApiLifecycleState;
+  disable_membership_notifications?: boolean;
+  background_url?: string;
+}
+
+export interface ResponseTemplate {
```

Comment:
for all the following types, I dislike the fact that we have everything in one and only file

Reply:
Yes, I know and I understand the initial motivation to split it into different files but I find it very difficult to assign every interface to a certain file. Some interfaces are used in different areas (files). I could try to assign them to other files but it partly feels quite arbitrary to do so.

Reply:
Alright, as discussed, I get your point and will try my best to put them into right files.